Makefile

```diff
@@ -377,6 +377,8 @@ tinygo-baremetal:
 .PHONY: smoketest
 smoketest:
 	$(TINYGO) version
+	# regression test for #2892
+	cd tests/testing/recurse && ($(TINYGO) test ./... > recurse.log && cat recurse.log && test $$(wc -l < recurse.log) = 2 && rm recurse.log)
```

Comment:
Try `cd tests/testing/recurse && ($(TINYGO) test ./... > recurse.log && cat recurse.log && test $$(wc -l < recurse.log | sed 's/ //g') = 2)`

Reply:
I think whitespace is stripped because I did not quote.

I verified that failure is not due to whitespace with this patch applied to your branch:
```
$ make smoketest
/Users/dkegel/go/bin/tinygo version
tinygo version 0.24.0-dev-2d619724 darwin/amd64 (using go version go1.18.1 and LLVM version 14.0.0)
# regression test for #2892
cd tests/testing/recurse && (/Users/dkegel/go/bin/tinygo test ./... > recurse.log && cat recurse.log && test $(wc -l < recurse.log) = 2 && rm recurse.log)
ok  	github.com/tinygo-org/tinygo/tests/testing/recurse/subdir	0.106s
make: *** [smoketest] Error 1
```

Note the single line in the output.

Have you tried this patch on your machine?

Reply:
Interesting, yeah I tried the test in my machine and after the fix I got 

```
➜  tinygo git:(fixes_three_dots) ✗ make smoketest TINYGO=~/Workspace/gource/bin/tinygo
~/Workspace/gource/bin/tinygo version
tinygo version 0.24.0-dev darwin/amd64 (using go version go1.17.2 and LLVM version 13.0.1)
# regression test for #2892
cd tests/testing/recurse && (~/Workspace/gource/bin/tinygo test ./... > recurse.log && cat recurse.log && test $(wc -l < recurse.log | sed 's/ //g') = 2)
ok      github.com/tinygo-org/tinygo/tests/testing/recurse      0.322s
ok      github.com/tinygo-org/tinygo/tests/testing/recurse/subdir       0.414s
```

But more interestingly is that I tried the patch just now without the space trimming and also got

```
➜  tinygo git:(fixes_three_dots) ✗ make smoketest TINYGO=~/Workspace/gource/bin/tinygo
~/Workspace/gource/bin/tinygo version
tinygo version 0.24.0-dev darwin/amd64 (using go version go1.17.2 and LLVM version 13.0.1)
# regression test for #2892
cd tests/testing/recurse && (~/Workspace/gource/bin/tinygo test ./... > recurse.log && cat recurse.log && test $(wc -l < recurse.log) = 2)
ok      github.com/tinygo-org/tinygo/tests/testing/recurse      0.073s
ok      github.com/tinygo-org/tinygo/tests/testing/recurse/subdir       0.139s
```

So I think my space trimming is not needed.

I wonder if you could open this PR against my branch to see how it behaves.

Reply:
Done.